### PR TITLE
console-sm: remove dependency list, replace with hint to sbom

### PR DESCRIPTION
**Pull request by ultraschuppi**
## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/reference/dependencies.md [hunk 1]
```diff
@@ -4173,463 +4173,7 @@ All of these libraries are required for core functionality.
...
-- [hoist-non-react-statics](https://github.com/mridgway/hoist-non-react-statics#readme) (BSD-3-Clause)
-- [rw](https://github.com/mbostock/rw) (BSD-3-Clause)
-- [source-map-js](https://github.com/7rulnik/source-map-js) (BSD-3-Clause)
-- [warning](https://github.com/BerkeleyTrue/warning) (BSD-3-Clause)
-- [director](https://github.com/flatiron/director#readme) (Unknown)
-- [dompurify](https://github.com/cure53/DOMPurify) ((MPL-2.0 OR Apache-2.0))
-- [robust-predicates](https://github.com/mourner/robust-predicates#readme) (Unlicense)
-- [tslib](https://www.typescriptlang.org/) (0BSD)
-- [wicg-inert](https://github.com/WICG/inert#readme) (W3C-20150513)
+Source Code and list of dependencies and their respective licenses (as CyclonDX SBOM) are provided [on-demand](dependency-request@camunda.com).
```
Comment (conceptualshark) [inline docs/reference/dependencies.md:4176]:
```suggestion
The Camunda 8 Self-Managed source code, dependency list, and dependency licenses are available as a CyclonDX SBOM [on-demand](dependency-request@camunda.com).
```

**Review by conceptualshark — COMMENTED** [review]
Do we have any internal documentation around this change?

**Comment by akeller** [comment]
I am not fully caught up on the SBOM conversation, so I won't be able to approve this yet.

File: docs/reference/dependencies.md (hunk 1 above)
Comment (MaxTru) [inline docs/reference/dependencies.md:4176]: Please change to make clear that SBOM and source code are two different assets

```suggestion
The source code and a CyclonDX SBOM, that includes a list of third party libraries used and their licenses, are provided [on-demand](dependency-request@camunda.com).
```

**Comment by akeller** [comment]
I edited the proposed commits just so we didn't accidentally introduce a typo, but can we make this even easier by adopting the same language used in the [already merged Web Modeler PR](https://github.com/camunda/camunda-docs/pull/4338)? 

> - **Dependencies:** SBOM CycloneDX files with up-to-date lists of third party libraries used and their licenses can be requested [on demand](mailto:dependency-request@camunda.com).
> - **Source code:** Access to source code is provided [on demand](mailto:dependency-request@camunda.com).

**Comment by MaxTru** [comment]
Superseded by https://github.com/camunda/camunda-docs/pull/4340

**Pull request closed without merging**